src/browser/sidebar.js

```diff
@@ -107,3 +108,13 @@ export const view = ({entries}, address) =>
       className: 'sidebar-toolbar'
     })
   ]);
+
```

Comment:
I do think that embedder of side bar should just wrap it in a container that is displayed or not based on the mode off the embedder. Dose that make sense ?